src/index.js

```diff
@@ -12,3 +20,16 @@ calcButtonNumbers.forEach(function(calcButtonNumber) {
     appendInput(calcInput, calcButtonNumber.innerText);
   });
 });
+
+operators.forEach(function(operator) {
+  operator.addEventListener("click", function() {
+    appendInput(calcInput, operator.innerHTML);
+  });
+});
+
+equal.addEventListener("click", () => {
+  let result = eval(calcInput.value.toString());
+  result = createDivWithContent(result);
```

Comment:
I would create a new variable. Otherwise, `result` could be a number or an element.
Maybe `const resultElement` or `const resultContainer`.